packages/core/src/tests/actions.test.ts

```diff
@@ -25,98 +19,138 @@ describe("Actions", () => {
                         content: { text: "Hi {{user1}}!", action: "wave" },
                     },
                 ],
+                [
+                    { user: "user1", content: { text: "Hey {{user2}}, how are you?" } },
+                    { user: "user2", content: { text: "I'm good {{user1}}, thanks!" } },
+                ],
             ],
-            similes: [],
-            handler: function (
-                _runtime: IAgentRuntime,
-                _message: Memory,
-                _state?: State,
-                _options?: { [key: string]: unknown },
-                _callback?: HandlerCallback
-            ): Promise<unknown> {
-                throw new Error("Function not implemented.");
-            },
-            validate: function (
-                _runtime: IAgentRuntime,
-                _message: Memory,
-                _state?: State
-            ): Promise<boolean> {
-                throw new Error("Function not implemented.");
-            },
+            similes: ["say hi", "welcome"],
+            handler: async () => { throw new Error("Not implemented"); },
+            validate: async () => { throw new Error("Not implemented"); },
         },
         {
             name: "farewell",
             description: "Say goodbye",
             examples: [
                 [
                     { user: "user1", content: { text: "Goodbye {{user2}}!" } },
+                    { user: "user2", content: { text: "Bye {{user1}}!" } },
+                ],
+            ],
+            similes: ["say bye", "leave"],
+            handler: async () => { throw new Error("Not implemented"); },
+            validate: async () => { throw new Error("Not implemented"); },
+        },
```

Comment:
We should probably implement these and test then properly in a follow up PR :)

Reply:
Can't wait! :)